app/core/services/product.service.ts

```diff
@@ -0,0 +1,57 @@
+
+import { CJsonApi } from './../models/jsonapi';
+import { Taxonomy } from './../models/taxonomy';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Product } from '../models/product';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+
+@Injectable()
+export class ProductService {
+	/**
+	 * Creates an instance of ProductService.
+	 * @param {HttpService} http
+	 *
+	 * @memberof ProductService
+	 */
+	constructor(
+		private http: HttpClient,
+
+	) { }
+	// tslint:disable-next-line:member-ordering
+	success: any;
+	// tslint:disable-next-line:member-ordering
+	error: any;
+	/**
+	 *
+	 *
+	 * @param {string} id
+	 * @returns {Observable<Product>}
+	 *
+	 * @memberof ProductService
+	 */
+
+	getProduct(id: string): Observable<Product> {
+		return this.http
+			.get<Product>(
+				`https://app.ofypets.com/api/v1/products/${id}?data_set=large`
```

Comment:
@jaypal412 hardcoded url's? @pkrawat1

Reply:
@ashish173 will add them to the backlog, as will require interceptor logic. @jaypal412  Will add that in next PR